CHANGELOG.md

```diff
@@ -2,6 +2,16 @@ postgresql Cookbook CHANGELOG
 =============================
 This file is used to list changes made in each version of the postgresql cookbook.
 
+v4.0.0
+-----
+* No more derived attributes in the attributes file
```

Comment:
Awesome. I'll update the item with your text. And yes there are a number of issues related. I will gather them up now.